recipe/build-runtime.sh

```diff
@@ -16,6 +16,12 @@ cp ./dotnet/dotnet* "${DOTNET_ROOT}"
 cp -r ./dotnet/shared/Microsoft.NETCore.App/ "${DOTNET_ROOT}/shared/Microsoft.NETCore.App/"
 cp -r ./dotnet/host/ "${DOTNET_ROOT}/host/"
 
+runtime_lib_dir=${DOTNET_ROOT}/shared/Microsoft.NETCore.App/${DOTNET_RUNTIME_VERSION}
+if [[ "${target_platform}" == "linux-"* ]]; then
+    # Hack to fix liblttng-ust dependency issues. liblttng-ust.so.0 is no longer supported.
+    patchelf --remove-needed liblttng-ust.so.0 ${runtime_lib_dir}/libcoreclrtraceptprovider.so
+fi
```

Comment:
I'm not entirely sure why we're making changes to the binaries downloaded from Microsoft.

A more descriptive comment saying *why* this change is being made (or a link to some description of the problem) wouldn't go amiss.

Reply:
Oh, has this been tested to still be an issue with `net9.0` & `aarch64`? I think this popped up initially for `net8.0`.

Reply:
Hmm, I would be on the opposite side of the fence with respect to your first comment, so I'll abstain to comment

The issue is not breaking the build, but it highlight that the `liblttng-ust.so.0` is requested by the binary library but missing in the environment, even though the lttng is installed, because it only provides `liblttng-ust.so.1` - It seemed safer to set the env variable and remove the 'needed' from the .so - Many other packagers of dotnet have done the same (I gather the info from the debian packager - I know nothing about dotnet - which relates to up above)